include/zephyr/drivers/stepper.h

```diff
@@ -117,12 +117,12 @@ typedef int (*stepper_enable_t)(const struct device *dev, const bool enable);
 typedef int (*stepper_move_by_t)(const struct device *dev, const int32_t micro_steps);
 
 /**
- * @brief Set the max velocity in microsteps per seconds.
+ * @brief Set the time interval between steps in nanoseconds.
  *
- * @see stepper_set_max_velocity() for details.
+ * @see stepper_set_step_interval() for details.
  */
-typedef int (*stepper_set_max_velocity_t)(const struct device *dev,
-					  const uint32_t micro_steps_per_second);
+typedef int (*stepper_set_step_interval_t)(const struct device *dev,
```

Comment:
sorry that this was not discovered in one of the earlier reviews. 
```suggestion
typedef int (*stepper_set_microstep_interval_t)(const struct device *dev,
```

Reply:
Done